packages/vda-web3-client/src/config.ts

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable prettier/prettier */
+import { Signer } from '@ethersproject/abstract-signer'
+
+import { BigNumber } from '@ethersproject/bignumber'
+import { Contract, ContractFactory } from '@ethersproject/contracts'
+import { JsonRpcProvider, Provider } from '@ethersproject/providers'
+import { ethers } from 'ethers'
```

Comment:
Not used?

Reply:
Used ethers dependency in vda-web3-client.